include/OpenSpaceToolkit/Astrodynamics/Flight/Maneuver.hpp

```diff
@@ -210,43 +206,21 @@ class Maneuver
     /// @param (optional) displayDecorators If true, display decorators
     void print(std::ostream& anOutputStream, bool displayDecorator = true) const;
 
-    /// @brief Create a maneuver from a tabulated dynamics with cols 1-3 being acceleration and col 4 being mass flow
-    /// rate
-    ///
-    /// @code{.cpp}
-    ///                  Maneuver maneuver = Maneuver::TabulatedDynamics(tabulated);
-    /// @endcode
-    ///
-    /// @param aTabulatedDynamics A TabulatedDynamics object
-    ///
-    /// @return A maneuver
-    static Maneuver TabulatedDynamics(const Tabulated& aTabulatedDynamics);
```

Comment:
Perhaps we could even change the `TabulatedDynamics` creation interface to allow creating tabulated dynamics from a State or something? That way you could still support this static method cause I thought we used it quite often, no?

Reply:
We don't use it at all anywhere.
It's not possible to create a Maneuver from just Tabulated dynamics, you would have to specify the position velocity, while a tabulated dynamics for thrusters typically only contains thrust acceleration and mass flow rate.